### PR TITLE
Build new page at /osm-hackfests

### LINKED ISSUE #68: Build new hackfests page /osm-hackfests
New webpage with title “Canonical Participation in OSM Hackfests”
[Copy doc](https://docs.google.com/document/d/1kpVxdUP4SxzJzmLmKMWFdWdPjQXHVYkhuBXfeUyZ-XE/edit#heading=h.d63qa3j9fmu4)

### PR DESCRIPTION
**Pull request by bethcollins92**
## Done

- Build new page at /osm-hackfests
- Add to nav 

## QA

- Check out this feature branch
- Run the site using the command `./run`
- View the site locally in your web browser at: http://0.0.0.0:8042/osm-hackfests
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check page against [copy doc](https://github.com/canonical-web-and-design/charmed-osm.com/issues/68) 

## Issue / Card

Fixes #68

### CODE REVIEW

File: templates/partial/_navigation.html [hunk 1]
```diff
@@ -7,5 +7,12 @@
         </a>
       </div>
     </div>
+    <nav class="p-navigation__nav">
+      <ul class="p-navigation__links" role="menu">
+        <li class="p-navigation__link" role="menuitem">
```
Comment (anthonydillon) [inline templates/partial/_navigation.html:12]: I think you should have an is-selected state

**Review by anthonydillon — APPROVED** [review]
I think it looks really good and works on small screens. One small comment in the code

File: templates/partial/_navigation.html (hunk 1 above)
Comment (bethcollins92) [reply templates/partial/_navigation.html:12]: Thanks, aah yes of course

**Comment by anasereijo** [comment]
LGTM 🥇

**Pull request merged**